qemu/tests/cfg/uefi_check_secure_MOR.cfg

```diff
@@ -0,0 +1,22 @@
+- uefi_check_secure_MOR:
+    only q35
+    only ovmf
+    only Win2016, Win10
+    type = uefi_check_secure_MOR
+    timeout = 360
+    start_vm = no
+    restore_ovmf_vars = yes
+    auto_cpu_model = no
+    cpu_model = host
+    check_secure_boot_enabled_cmd = 'powershell -command "Confirm-SecureBootUEFI"'
+    dgreadiness_tool = '/home/dgreadiness.zip'
+    src_path = '/home/dgreadiness_v3.6'
+    dst_path = "C:\dgreadiness"
+    dgreadiness_tool_path = "cd ${dst_path}\dgreadiness_v3.6"
+    download_cmd = 'wget https://download.microsoft.com/download/B/D/8/BD821B1F-05F2-4A7E-AA03-DF6C4F687B07/dgreadiness_v3.6.zip -O ${dgreadiness_tool}'
```

Comment:
The link might be breakable, how about getting a copy and putting it into deps dir?